frontend/lib/api-interceptor.ts

```
@@ -0,0 +1,310 @@
+   * 执行带拦截器的请求
+   */
+  static async request<T = unknown>(
+    config: RequestConfig,
+  ): Promise<ApiResponse<T>> {
+    let attempt = 0;
+    let lastError: Error | null = null;
+
+    while (attempt < this.MAX_RETRY_ATTEMPTS) {
+      try {
+        return await this.executeRequest<T>(config);
+      } catch (error) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)